tsl/src/nodes/decompress_chunk/exec.c

```diff
@@ -391,6 +216,26 @@ constify_tableoid(List *node, Index chunk_index, Oid chunk_relid)
 	return node;
 }
 
+pg_attribute_always_inline static TupleTableSlot *
```

Comment:
You mentioned the idea behind the small functions and the inlining when we discussed the PR. Could you add this information also as a comment here?

Reply:
It has a comment about this before the implementation.